content/tutorials/r-core/4-plotting/index.rmd

```diff
@@ -70,18 +70,27 @@ ggplot(nhanes, aes(x = Age, y = Weight))
   * Axes are set up the way we'd expect, and seem to have sensible values
   * Why is nothing on this graph yet? we haven't put any graphic actually on the axes yet
 
-**We need to tell the `ggplot()` call what kind of graphic to put on the axis**
+**We need to tell the `ggplot()` call what kind of graphic to put on the axis.**
 
   * A lot of the time, the syntax is `geom_[something]`
 
 # Example Plot Types
 
 ## Scatter plots
 
-Let's do scatter with `geom_point()` first.
+Let's create a scatter plot using `geom_point()` first. The `ggplot()` function generates plots in layers, from "bottom" to "top", and each layer of the plot is separated by the `+` sign.
 
 ```{r}
+# Although you can keep the code all on a single line, like this:
 ggplot(nhanes, aes(x = Age, y = Weight)) + geom_point()
+
+# As you add more and more layers, it can be helpful to add a
+# line break after each `+` sign, like this:
+ggplot(nhanes, aes(x = Age, y = Weight)) +
+  geom_point()
+
```

Comment:
Big +1 to line breaking after the + for each ggplot layer! Readability for the win!